broker-core/src/main/java/io/zeebe/broker/workflow/processor/instance/CreateWorkflowInstanceOnStartEventHandler.java

```diff
@@ -33,6 +33,8 @@
 public class CreateWorkflowInstanceOnStartEventHandler
     implements BpmnStepHandler<ExecutableFlowElementContainer> {
 
+  public static final String NO_WORKFLOW_FOUND_MESSAGE =
+      "Expected to create an instance of workflow with key '%d', but no such workflow was found";
```

Comment:
These not found messages are also really often repeated.. Ah ok you will tackle this with the follow up issue :+1:

Reply:
Yes, there's a lot of overlap/duplication, and I already thought a bit about it, I think with rejections there's a clear easy API (i.e. using the command control to reject things in a structured way), with the more generic `ErrorResponse` I'm not sure yet, but yeah, in the follow up issue this should all be addressed.

I think once we have a first end-to-end iteration (i.e. broker, gateway, client), it might be clearer what we want/how best to do it.